extensions/BMO/Extension.pm

```diff
@@ -713,6 +713,18 @@ sub bug_format_comment {
         }
     });
 
+    # link github pull requests and issues
+    push (@$regexes, {
+        match => qr#\b([\S]+\/[\S]+\#[0-9]+)\b#mx,
```

Comment:
@upsuper I guess, it should also contain a space before and after, so that links like these "sample/to/repo#num" don't link?